src/Sylius/Component/Core/Test/Services/DefaultCountriesFactoryInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Test\Services;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface DefaultCountriesFactoryInterface
```

Comment:
I think question is - do we really need a separate service **for now**? The only feature we're trying to achieve is configuring store shipping-available countries. There is no need for provider described by you, but also I suppose there is no need for factory I've implemented :smile:
So my idea is to get rid of any additional service and configure shipping countries for store in ``ChannelContext`` (or event separate ``CountryContext``?). @pamil WDYT?